File: patterns/c-patterns/pattern20.c
// Pattern 20: ...

// A****
// BB***
// CCC**
// DDDD*
// EEEEE

#include <stdio.h>
int main()
{
    int i, j;
    int n = 5;
    for (i = 0; i < n; i++)
    {
        for (j = 0; j < n; j++)
        {
            if (j > i)
                printf("*");
            else
                printf("%c", i + 65);
        }
        printf("\n");
    }
    return 0;
}

File: patterns/python-patterns/pattern20.py
# Pattern 20: ...

# A****
# BB***
# CCC**
# DDDD*
# EEEEE

n = 5

for i in range(n):
    for j in range(n):
        if j > i:
            print("*", end="")
        else:
            print(chr(i + 65), end="")
    print()
